### PR TITLE
Add PostgreSQL configuration example

### PR DESCRIPTION
**Pull request by dacostafilipe**
Example of the configuration we use internally for using PostgreSQL with `ddev`

### CODE REVIEW

File: docker-compose-services/postgres/docker-compose.postgres.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - POSTGRES_DB=db
+    volumes:
+      - ".:/mnt/ddev_config"
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: $DDEV_APPROOT
+      com.ddev.app-url: $DDEV_URL
+  web:
+    links:
+      - postgres:postgres
```
Comment (rfay) [inline docker-compose-services/postgres/docker-compose.postgres.yaml]: Needs a newline at the end. No biggie.

**Comment by rfay** [comment]
```
$ ddev pgsql_export
WARNING: The DDEV_URL variable is not set. Defaulting to a blank string.
bash: /mnt/ddev_config/import-db/postgresql.db.sql: No such file or directory
Failed to run pgsql_export : exit status 1
```

I don't think the import-db directory is ever there in ddev any more, it's from long-ago versions of ddev?

DDEV_URL is obsolete, I'll comment there.

File: docker-compose-services/postgres/docker-compose.postgres.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    environment:
+      - POSTGRES_PASSWORD=db
+      - POSTGRES_USER=db
+      - POSTGRES_DB=db
+    volumes:
+      - ".:/mnt/ddev_config"
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: $DDEV_APPROOT
+      com.ddev.app-url: $DDEV_URL
```
Comment (rfay) [inline docker-compose-services/postgres/docker-compose.postgres.yaml]: This line is obsolete, just remove it (com.ddev.app-url: $DDEV_URL)

File: docker-compose-services/postgres/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+## PostgreSQL
+
+Using PostgreSQL container with PostGIS support provided by [mdillon/postgis
+](https://hub.docker.com/r/mdillon/postgis).
```
[the remark this answers is not in the document]
Comment (dacostafilipe) [reply docker-compose-services/postgres/README.md]: In most of our project, we need some kind of spacial search (locations around you, locations in polygon, distance between locations, ... ) and it's really easy and fast with PostGIS. Until activated, there's zero performance penalty.

File: docker-compose-services/postgres/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+``` 
+
+### Import / Export
+
+Two new `ddev` commands are provided:
+
+- `ddev pgsql_export` : Use `pg_dump` to export `db` to `import-db/postgresql.db.sql` 
+- `ddev pgsql_import` : Use `pgsql` to import `import-db/postgresql.db.sql` into `db`
+
+To import/export the `db` table automatically add the following hooks to your `config.yaml`:
```
[the remark this answers is not in the document]
Comment (dacostafilipe) [reply docker-compose-services/postgres/README.md]: I don't remember why we did it that way, will look into this.

**Comment by dacostafilipe** [comment]
> I don't think the import-db directory is ever there in ddev any more, it's from long-ago versions of ddev?

1.9 I think.

> Wouldn't it be better to export to stdout though? Let people do with that what they will?

That's how the script worked at the beginning. But I had issues with the import part where I did not find an easy way to pass stdin into `su -c`. I will rewrite the import to create a temp file with the stdin instead.

File: docker-compose-services/postgres/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (dacostafilipe) [reply docker-compose-services/postgres/README.md]: Well, seems like it was just an example of `calculated laziness`. We needed the import/export commands and they "just worked" for us when restarting projects.

**Comment by dacostafilipe** [comment]
The changes to the commands do not seems to work correctly. Executing them manually works, but running them in the hooks does not. Could be stdin/stdout issue as it worked before the change.

**Comment by dacostafilipe** [comment]
I reverted the commands for now.

File: docker-compose-services/postgres/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rfay) [reply docker-compose-services/postgres/README.md]: I was saying to say "why" in the README or docker-compose file. So people can understand why you made the choice (and maybe understand whether they should also be using this, or maybe the library version)

**Comment by rfay** [comment]
@dacostafilipe I made some minor changes to the readme and the commands in https://github.com/drud/ddev-contrib/pull/7/commits/15d9701816867361c949858b1b0fd14952c008a9 - if this is OK with you and you're ready for this to be pulled I'll pull it.

**Comment by dacostafilipe** [comment]
I'm okay with the changes. Added those to my projects.

**Comment by rfay** [comment]
https://github.com/drud/ddev/pull/1819 will add the ability to omit the db container, and it will be good to add that here, it should make things less confusing for people.

**Comment by steffenr** [comment]
We are using pgsql for one of our clients in a ddev setup. Usually we "fire" some post-import-db Commands after DB Import. But those commands are not fired after importing the pgsql. 

Is this still possible?

**Comment by dacostafilipe** [comment]
@steffenr 

Yes, it works on our postgresql project. 

Our hooks look like this:
`
hooks:
  pre-start:
    - exec-host: bash .ddev/run-os-export.sh
  pre-stop:
    - exec-host: ddev pgsql_export
  post-start:
    - exec-host: ddev pgsql_import
    - exec: ./console.php queue:worker --queueType=realtime
`

**Comment by steffenr** [comment]
@dacostafilipe 
You are using the global hooks - we had the *post-db-import* hook for firing our commands. Those commands are triggered every ddev start/ stop
In our case we only want to reset password/ activate stage_file_proxy after importing the db.

**Comment by rfay** [comment]
Since importing doesn't use `ddev import-db` you can't use the post-import-db hooks. But you *can* use a post-start hook that runs a script that's smart enough to check to see if stage_file_proxy is already activated, for example, and if not, activate it. Or use a flag in the database or on disk.

**Pull request merged**